docs/source/en/model_doc/intern_image.mdx

```diff
@@ -0,0 +1,86 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# internimage
```

Comment:
In the paper, it's capitalized differently - InternImage

Reply:
fixed with [ae189ec](https://github.com/huggingface/transformers/pull/23782/commits/ae189ec5739f6595146e05abce4dfdd11c940f8d)